### PR TITLE
feat: add suites icons and order them alphabetically

### PR DESCRIPTION
**Pull request by gabyzif**
This PR adds the suites icons showed in [this](https://www.figma.com/design/XqsAQacU5VCAxESlgWe6ix/Unify-M1.5?node-id=3070-8751&m=dev) figma and it orders them alphabetically so it's easier to look them up.

Screenshot 2024-05-20 at 7 52 41 PM

Left image: the icons in the aquarium
Right image: the icons in the Figma

Each icon is hiighlighted with the same color to ensure that they exist in the aquarium.

Note: the enhancement icon is different because the one in the figma is an old version, design gave me the new version a couple of weeks ago.
Identity and the userProfile are not signed off with design.

Note2: the following Icons were already there but with another name:

trends: chartline
setup: wrench
audiences: users
journeys: split
privacy settings: shieldKeyhole

Storybook Link: [Current branch](https://65aec861fa4adbf0708aebd0-wxeghnyxmv.chromatic.com/?path=/docs/aquarium-general-icons--documentation )

Main link in the stories:[main]( https://65aec861fa4adbf0708aebd0-ghamqwxjmh.chromatic.com/?path=/story/aquarium-general-icons--icon-table )

### CODE REVIEW

**Review by tibuurcio — APPROVED** [review]
Nice work on the sorting :)

Confirmed with @gabyzif this is not a breaking change since we didn't change any names. Just reordering and adding new icons.

**Comment by jared-dickman** [comment]
lets make sure all new icons are up-to-code, including `currentColor` for example 
https://github.com/mParticle/aquarium/blob/bc36ba78e5e14a248c871c0ff0a02ccbd964bc21/src/assets/readme.md#L10

File: src/assets/svg/mp_pm_lt_user.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg" fill="currentColor">
+<g id="square-user">
```
Comment (jared-dickman) [inline src/assets/svg/mp_pm_lt_user.svg:2]: curious this one has a wrapping `g` with `id` where the other assets dont 💭

Comment (gabyzif) [reply src/assets/svg/mp_pm_lt_user.svg:2]: this are the new final icons exported from Caro :)

Comment (gabyzif) [reply src/assets/svg/mp_pm_lt_user.svg:2]: if we don't want the Id, I can add a part into the prettified script to remove it

Comment (jared-dickman) [reply src/assets/svg/mp_pm_lt_user.svg:2]: id say this empty group with id would be good to remove

**Comment by gabyzif** [comment]
waiting until Designs tells us what we do with the 2 uncurated icons: 
Screenshot 2024-05-20 at 11 36 32 PM

**Pull request closed without merging**